tracing-attributes/src/lib.rs

```diff
@@ -187,7 +185,7 @@ pub fn instrument(args: TokenStream, item: TokenStream) -> TokenStream {
     let level = level(&args);
     let target = target(&args);
 
-    quote_spanned!(call_site=>
+    quote!(
```

Comment:
I think propagating the span of the original function is important, so that any errors are displayed correctly. Is there a way to do that using `syn::spanned::Spanned`, so that we don't need to import the `Span` type explicitly?

I believe that we can use [the `Spanned` impl for `ItemFn`](https://docs.rs/syn/1.0.3/syn/struct.ItemFn.html) for this.

Reply:
I may be misunderstanding the documentation, but [this] seems to indicate that `quote!(…)` does the same thing as `quote_spanned!(Span::call_site()=> …)`

[this]: https://docs.rs/quote/1.0.2/quote/macro.quote.html#hygiene

Reply:
Oh, I think I missed that in the docs, that's great. Nevermind my previous comment!